config/formats.js

```diff
@@ -378,19 +378,18 @@ let Formats = [
 		},
 		ruleset: ['HP Percentage Mod', 'Cancel Mod'],
 		banlist: [
-			'Aegislash', 'Aerodactylite', 'Aggronite', 'Aguav Berry', 'Ampharosite', 'Assault Vest', 'Berry', 'Berry Juice', 'Black Sludge',
+			'Aegislash', 'Aerodactylite', 'Aggronite', 'Aguav Berry', 'Ampharosite', 'Assault Vest', 'Berry', 'Berry Juice', 'Berserk Gene', 'Black Sludge',
 			'Blastoisinite', 'Blazikenite', 'Charizardite X', 'Charizardite Y', 'Cheek Pouch', 'Cursed Body', 'Desolate Land', 'Diancite',
-			'Drizzle ++ Dry Skin', 'Drizzle ++ Rain Dish', 'Doublade', 'Enigma Berry', 'Figy Berry', 'Fur Coat', 'Galladite', 'Garchompite',
+			'Doublade', 'Dry Skin', 'Enigma Berry', 'Figy Berry', 'Fur Coat', 'Galladite', 'Garchompite',
```

Comment:
Why is the spacing so inconsistent??????

Reply:
Don't worry! I'm reordering all of this as we speak!